samples/instanceadmin/instanceadmin.py

```diff
@@ -1,259 +1,270 @@
+        print("Created development instance: {}".format(instance_id))
+    else:
+        print("Instance {} already exists.".format(instance_id))
+
+    # [END bigtable_create_dev_instance]
+
+
+def delete_instance(project_id, instance_id):
+    """Delete the Instance
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+    """
+
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    # [START bigtable_delete_instance]
+    print("\nDeleting instance")
+    if not instance.exists():
+        print("Instance {} does not exist.".format(instance_id))
+    else:
+        instance.delete()
+        print("Deleted instance: {}".format(instance_id))
+    # [END bigtable_delete_instance]
+
+
+def add_cluster(project_id, instance_id, cluster_id):
+    """Add Cluster
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+
+    :type cluster_id: str
+    :param cluster_id: Cluster id.
+    """
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+
+    location_id = "us-central1-a"
+    serve_nodes = 1
+    storage_type = enums.StorageType.SSD
+
+    if not instance.exists():
+        print("Instance {} does not exist.".format(instance_id))
+    else:
+        print("\nAdding cluster to instance {}".format(instance_id))
+        # [START bigtable_create_cluster]
+        print("\nListing clusters...")
+        for cluster in instance.list_clusters()[0]:
+            print(cluster.cluster_id)
+        cluster = instance.cluster(
+            cluster_id,
+            location_id=location_id,
+            serve_nodes=serve_nodes,
+            default_storage_type=storage_type,
+        )
+        if cluster.exists():
+            print("\nCluster not created, as {} already exists.".format(cluster_id))
+        else:
+            cluster.create()
+            print("\nCluster created: {}".format(cluster_id))
+        # [END bigtable_create_cluster]
+
+
+def delete_cluster(project_id, instance_id, cluster_id):
+    """Delete the cluster
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+
+    :type cluster_id: str
+    :param cluster_id: Cluster id.
+    """
+
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    cluster = instance.cluster(cluster_id)
+
+    # [START bigtable_delete_cluster]
+    print("\nDeleting cluster")
+    if cluster.exists():
+        cluster.delete()
+        print("Cluster deleted: {}".format(cluster_id))
+    else:
+        print("\nCluster {} does not exist.".format(cluster_id))
+
+    # [END bigtable_delete_cluster]
+
+
+if __name__ == "__main__":
```

Comment:
These samples are very old so they still have this CLI runner. Which might be good to remove and bring these into the newer format?

I think because this is a sample vs snippets they can stay all in one file, but this is something I need to think about from a Bigtable perspective rather than just for this PR

Reply:
It might make sense to make a separate issue with its own PR for this, since it's wider in scope than just this one test.

Reply:
sounds good